lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -314,7 +321,18 @@ defmodule ExUnit.Case do
       Module.put_attribute(mod, :ex_unit_test_names, Map.put(test_names, name, true))
     end
 
-    Module.delete_attribute(mod, :tag)
+    Enum.each [:tag | test_attributes], fn(attribute) ->
+      Module.delete_attribute(mod, attribute)
+    end
+  end
+
+  @doc """
+  """
+  defmacro register_attribute(mod, new, opts \\ []) when is_atom(new) do
```

Comment:
I was using the same function signature as `Module.register_attribute/3` https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/module.ex#L969